src/RageTimer.h

```diff
@@ -20,8 +20,6 @@ class RageTimer
 
 	/* Time between last call to GetDeltaTime() (Ago() + Touch()): */
 	float GetDeltaTime();
-	/* (alias) */
-	float PeekDeltaTime() const { return Ago(); }
```

Comment:
I wonder why it existed in the first place, but seems fine to remove.

Reply:
Even the original commit doesnt make it obvious:

https://github.com/itgmania/itgmania/commit/fbcd8042436d3754f9cb368f39741a507e43f392#diff-bba0da2f78db9e0847412f4946291ba1fdc3f239fe5d74b59a6e5ff3a8069cc5

Probably just for simple backwards compat